src/logic/jwt.js

```diff
@@ -0,0 +1,25 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+
+import jwt from 'jsonwebtoken';
+import { config } from '../config';
+
+type Payload = {|
+  +profileToken: string,
+|};
+
+class JwtConfigurationError extends Error {
+  name = 'JwtConfigurationError';
+  status = 500;
+  expose = true; // The message will be exposed to users
+}
+
+export function getToken(payload: Payload): string {
```

Comment:
It would be nice to write a small justification for how and why we are using JWT here. Plus it could be helpful to link to some of the literature we reviewed:

https://auth0.com/learn/json-web-tokens/
https://www.chosenplaintext.ca/2015/03/31/jwt-algorithm-confusion.html
https://auth0.com/blog/a-look-at-the-latest-draft-for-jwt-bcp/

Reply:
Also to be clear, I would save more detailed documentation for a future PR.